src/charts/candle/HoverTrap/index.web.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { parse } from 'react-native-redash';
+
+import { LineChartDimensionsContext } from '../../line/Chart';
```

Comment:
Is this meant to reference the candle implementation?

Reply:
Oh whoops this was unfinished. We should just remove the candle one for this PR.

Reply:
I don’t actually use the candle atm so I haven’t tested there. But I assume the same logic should apply.

Reply:
Yep. I am happy to leave it out for candle for the moment. Do you think we could add some quick docs to the readme for this too?

Reply:
Yeah I can add some docs and fix the conflicts when I have a second soon.

also, I just realized that the readme has messed up tables currently (scroll down to see them)